packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -121,11 +122,10 @@ export default async function pluginContentBlog(
           blogListPaginated: [],
           blogTags: {},
           blogTagsListPath,
-          blogTagsPaginated: [],
         };
       }
 
-      // Colocate next and prev metadata.
+      // Collocate next and prev metadata.
```

Comment:
I don't mind but afaik both are valid?

Reply:
The spell checker in my VS Code spotted it as a mistake and recommended collocate. The dictionary gives collocate and co-locate as valid.